src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -50,7 +50,47 @@
 import teammates.storage.api.StudentsDb;
 
 public class FeedbackSessionsLogic {
-
+    
+    private static final String ASSUMPTION_FAIL_DELETE_INSTRUCTOR = "Fail to delete instructor respondant for ";
+    private static final String ASSUMPTION_FAIL_RESPONSE_ORIGIN = "Client did not indicate the origin of the response";
+    private static final String ASSUMPTION_FAIL_RESPONSES_ORIGIN = "Client did not indicate the origin of the responses";
+    private static final String ERROR_NUMBER_OF_RESPONSES_EXCEEDS_RANGE = "Number of responses exceeds the limited range";
+    private static final String ERROR_SENDING_EMAILS = "Error while sending emails :";
+    private static final String ERROR_SESSION_UNPUBLISH_ALREADY = "Session is already unpublished.";
+    private static final String ERROR_SESSION_UNPUBLISH_NON_EXISTENT = "Trying to unpublish a non-existent session.";
+    private static final String ERROR_SESSION_UNPUBLISH_PRIVATE = "Private session can't be unpublished.";
+    private static final String ERROR_SESSION_PUBLISH_ALREADY = "Session is already published.";
+    private static final String ERROR_SESSION_PUBLISH_NON_EXISTENT = "Trying to publish a non-existent session.";
+    private static final String ERROR_SESSION_PUBLISH_PRIVATE = "Private session can't be published.";
+    private static final String KEY_FROM_SECTION = "fromSection";
+    private static final String KEY_IN_SECTION = "inSection";
+    private static final String KEY_IS_INCLUDE_RESPONSE_STATUS = "isIncludeResponseStatus";
+    private static final String KEY_QUESTION_ID = "questionId";
+    private static final String KEY_RANGE = "range";
+    private static final String KEY_SECTION = "section";
+    private static final String KEY_TO_SECTION = "toSection";
+    private static final String KEY_VIEW_TYPE = "viewType";
+    private static final String VALUE_FALSE = "false";
+    private static final String VALUE_TRUE = "true";
+    private static final String STRING_PERIOD = ".";
+    private static final String STRING_COLON = ": ";
+    private static final String ERROR_SESSION_DOES_NOT_EXIST_STRING_FORMAT = "Trying to %s a feedback session that does not exist%s";
+    private static final String ERROR_SESSION_DOES_NOT_EXIST_EXTRA_PARAM_STRING_FORMAT = "Trying to %s feedback session%s that does not exist.";
```

Comment:
The error here is that the student or course does not exist, so maybe we can rephrase it to something like `"Error getting feedback session(s): %s does not exist."`, where `%s` may be `"student"` or `"course"`